### PR TITLE
GAC minor revisions

### LINKED ISSUE #683: Call GAC property variables from ion_DSPMDE_prop_pack and other GAC changes
Move any properties constructed in gac.py to corresponding property package model, DSPMDE_prop_pack. This includes:

High Priority:
- [ ] allow for multi-species adsorption using the ideal adsorbed solution theory (IAST)
- [x] Remove redundant equation with circular reference causing failure depending on set of fixed variables
- [ ] call molecular diffusion coefficient from prop pack
- [ ] embed molar volume calculation

Low Priority:
- [x] update lb of `gac.equil_conc`
- [ ] dynamic sf for `gac.equil_conc` as f(`inlet_conc`) 
- [x] correct line 1144 for reporting of molar flowrate to port
- [x] correct typo in Schmidt number doc string
- [ ] implement check for `gac.length_mtz` size being approximately ≤ `gac.bed_length`
- [x] Add bed diameter as relevant sizing variable

### PR DESCRIPTION
**Pull request by hunterbarber**
## Fixes/Resolves:
Pushing minor revisions that were backlogged in #683

## Summary/Motivation:
Advancing GAC model development

## Changes proposed in this PR:
- Addition of bed diameter for optional size spec
- Grouping variables in test file for more clarity
- Minor formatting changes
- Lower bounds for some variables to handle treatment of ppm-ppt concentrations

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: watertap/unit_models/gac.py [hunk 1]
```diff
@@ -782,6 +790,14 @@ def eq_bed_volume(b):
                 == b.bed_volume
             )
 
+        @self.Constraint(doc="Adsorber bed area")
+        def eq_bed_area(b):
+            return b.bed_area == np.pi * (b.bed_diameter**2) / 4
```
Comment (bknueven) [inline watertap/unit_models/gac.py]:
```suggestion
            return b.bed_area == math.pi * (b.bed_diameter**2) / 4
```

File: watertap/unit_models/gac.py [hunk 2]
```diff
@@ -11,9 +11,9 @@
 #
 ###############################################################################
 
+import numpy as np
```
Comment (bknueven) [inline watertap/unit_models/gac.py]:
```suggestion
import math
```

**Review by bknueven — COMMENTED** [review]
Better to use the build-in math module when we can.

**Review by kurbansitterley — APPROVED** [review]
LGTM

**Comment by kurbansitterley** [comment]
> Better to use the build-in math module when we can.

What about ``from idaes.core.util.constants import Constants`` for pi? This is what I use in ion exchange model.

**Review by bknueven — APPROVED** [review]

**Comment by hunterbarber** [comment]
> 

I wasn't aware of either `math` or `idaes.core.util.constants.Constants`. Not sure if there is a standard or if we necessarily need a standard for this. It's up to discussion.

**Comment by andrewlee94** [comment]
@hunterbarber The IDAES standard is to use the `Constants` module to ensure consistency.

**Comment by hunterbarber** [comment]
> @hunterbarber The IDAES standard is to use the `Constants` module to ensure consistency.

Thanks for clarifying, I'll make the edit now before merge.

**Pull request merged**